frontend/src/App.js

```diff
@@ -1,11 +1,35 @@
-import React from 'react'
+import React from 'react';
+import styled from 'styled-components/macro';
+import HomePage from './homePage/HomePage';
+import MenuPage from './menuPage/MenuPage';
+
+import AddFavoritePage from './addFavoritePage/AddFavoritePage';
+import RemoveFavoritePage from './removeFavoritePage/RemoveFavoritePage';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 function App() {
   return (
-    <div>
-      <p>All Your Favorites</p>
-    </div>
+    <>
+      <PageLayout>
+        <Router>
```

Comment:
Du könntest den Router auch in der index.js um alles drum herum legen

Reply:
bzw hier einfach raus nehmen. Du brauchst nur switch in der App.js

Reply:
Danke für dein review und für den Hinweis. Hatte ich urspr. auch so, aber dann lief der Test nicht durch. Habs auch jetzt noch mal ausprobiert, aber bekomme immer folgenden Error: 

"Error: Invariant failed: You should not use <Switch> outside a <Router>"

Vielleicht ist noch irgendwas anderes falsch, wüsste aber nicht wo...